nixos/modules/services/monitoring/prometheus/exporters/nut.nix

```diff
@@ -36,6 +36,17 @@ in
         provisioned outside of Nix store.
       '';
     };
+    nutVariables = mkOption {
+      type = types.listOf types.str;
+      default = [ ];
+      description = ''
+        List of NUT variable names to monitor.
+
+        If no variables are set, all numeric variables will be exported automatically.
```

Comment:
Hi, I set up nut_exporter recently and learned I needed `nutVariables = [ "" ]` (list with empty string) to trigger upstream behaviour of exporting all metrics. (Before I did that my grafana dashboard had missing data.)

The wording of the option description here sounds like this behaviour is default (export all variables), but it seems not to be the case.

Should we change the default to export all? Or update the docstring? Or contact upstream about why exporting everything isn't enabled by default?